fiftyone/utils/huggingface.py

```diff
@@ -109,6 +111,10 @@ def push_to_hub(
     else:
         tags = _get_dataset_tags(dataset)
 
+    # do this now in case HF login fails before we do anything expensive
+    hf_username = hfh.whoami(token=token)["name"]
```

Comment:
needed to pass `token` here or else `HF_TOKEN` is strictly required

Reply:
@jacobmarks FYI I had an older version of `huggingface_hub` installed that didn't seem to support the `HF_TOKEN` environment variable. Can you track down the min version that supports it and update the `ensure_package()` call above to let the user know the min version that they need?

Reply:
Looks like this is where they added support in Google Colab at least: https://github.com/huggingface/huggingface_hub/pull/1895
👉 https://github.com/huggingface/huggingface_hub/releases/tag/v0.20.0
so `>=0.20.0`